File: webchat.frontend/src/app/shared/chatMessage.ts
export class ChatMessage {

    constructor(userId: string, userName: string, avatarId: number, message: string){
        this.userId = userId;
        this.userName = userName;
        this.avatarId = avatarId;    
        this.message = message;
    }

    userId: string;
    userName: string;
    avatarId: number;
    message: string;
}

File: webchat.backend/webchat.crosscutting/Domain/ChatMessage.cs
namespace webchat.crosscutting.Domain;

public sealed class ChatMessage
{
    public string UserId { get; set; }
    public string UserName { get; set; }
    public int AvatarId { get; set; }
    public string Message { get; set; }
    public DateTime SentAt { get; set; } = DateTime.Now;
}